src/styles/RepositoryAvatar.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import {borderRadius} from "./variables";
+
+const RepositoryAvatar = styled.img`
+  margin-right: 10px !important;
+  margin-bottom: -10px;
+  display:inline;
+  object-fit: cover;
+  height: 40px;
+  width: 40px;
```

Comment:
```suggestion
  height: ${gutter.medium};
  width: ${gutter.medium};
```

Use `${gutter.medium}`

Reply:
Done.